deployments/helm/hephaestus/templates/buildkit/configmap.yaml

```diff
@@ -27,4 +27,15 @@ data:
         all = true
         keepBytes = {{ .gcKeepStorage | int64 }}
       {{- end }}
+
+    {{- range $server, $opts := $.Values.registries }}
+    [registry."{{ $server }}"]
+      {{- with $opts.http }}
+      http = {{ . }}
+      {{- end }}
+
+      {{- with $opts.insecure }}
+      insecure = {{ . }}
+      {{- end }}
+    {{- end }}
```

Comment:
I know this will configure "pull" from insecure/http, but does it also handle "push", or do we also need to pass that information to the buildkit client?

Reply:
Confirmed it works for both by setting the internal DR to `http` and got back a 400.